x/wasm/keeper/querier_test.go

```diff
@@ -736,19 +736,28 @@ func TestQueryPinnedCodes(t *testing.T) {
 
 	q := Querier(keeper)
 	specs := map[string]struct {
-		srcQuery   *types.QueryPinnedCodesRequest
-		expCodeIDs []uint64
-		expErr     error
+		paginationTotal         uint64
+		paginationNextKeyLength uint64
+		codeIdsLength           uint64
```

Comment:
I think the codeIdsLength is duplicated with expCodeIDs. If codeIdsLength is wrong, expCodeIDs must not be equal.